code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -476,13 +476,35 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	description = "100 proof cinnamon schnapps, made for alcoholic teen girls on spring break."
 	color = "#FFFF91" // rgb: 255, 255, 145
 	boozepwr = 25
-	quality = DRINK_VERYGOOD
+	quality = DRINK_NICE  // teenager girls think it's DRINK_VERYGOOD
 	taste_description = "burning cinnamon"
 	glass_icon_state = "goldschlagerglass"
 	glass_name = "glass of goldschlager"
 	glass_desc = "100% proof that teen girls will drink anything with gold in it."
 	shot_glass_icon_state = "shotglassgold"
 
+/datum/reagent/consumable/ethanol/goldschlager/expose_mob(mob/living/exposed_mob, methods=TOUCH, reac_volume)
+	// Reset quality if someone's sharing the bottle
+	quality = initial(quality)
+
+	if(ishuman(exposed_mob))
+		var/mob/living/carbon/human/human = exposed_mob
+		// tgstation13 does not endorse underage drinking. laws may vary by your jurisdiction.
+		if(human.age >= 13 && human.age <= 19 && human.gender == FEMALE)
```

Comment:
the age literally cannot be below 19 in game, no point in doing anything other than == 19

Reply:
Isn't the minimum 17?

Reply:
right, i forgot

Reply:
So it should be 17 to 19 then.

Reply:
I'm not sure what we gain by specifying a more narrow range; and we definitely dilute the joke.

Reply:
Isn’t the min age a config? Should be fine to put whatever range in there

Reply:
Future proofing for whatever method of lowering someone's age that may exist. Also, what if podpeople have their age start at 0? What if we measure (gender-changed) Ian's age as rounds alive, and then he decided to booze it up?